content/docs/setup/kubernetes/install/kubernetes/index.md

```diff
@@ -102,16 +102,23 @@ $ kubectl apply -f install/kubernetes/istio-demo-auth.yaml
+    istio-egressgateway      ClusterIP      172.21.246.42    <none>          80/TCP,443/TCP                                                                                                            2m
+    istio-galley             ClusterIP      172.21.193.181   <none>          443/TCP,9093/TCP                                                                                                          2m
+    istio-ingressgateway     LoadBalancer   172.21.252.83    52.116.22.244   80:31380/TCP,443:31390/TCP,31400:31400/TCP,15011:30981/TCP,8060:30852/TCP,853:32192/TCP,15030:32267/TCP,15031:31601/TCP   2m
+    istio-pilot              ClusterIP      172.21.24.184    <none>          15010/TCP,15011/TCP,8080/TCP,9093/TCP                                                                                     2m
+    istio-policy             ClusterIP      172.21.49.95     <none>          9091/TCP,15004/TCP,9093/TCP                                                                                               2m
+    istio-sidecar-injector   ClusterIP      172.21.54.153    <none>          443/TCP                                                                                                                   2m
+    istio-telemetry          ClusterIP      172.21.183.182   <none>          9091/TCP,15004/TCP,9093/TCP,42422/TCP                                                                                     2m
+    jaeger-agent             ClusterIP      None             <none>          5775/UDP,6831/UDP,6832/UDP                                                                                                2m
+    jaeger-collector         ClusterIP      172.21.112.197   <none>          14267/TCP,14268/TCP                                                                                                       2m
+    jaeger-query             ClusterIP      172.21.189.44    <none>          16686/TCP                                                                                                                 2m
+    kiali                    ClusterIP      172.21.231.125   <none>          20001/TCP                                                                                                                 2m
+    prometheus               ClusterIP      172.21.53.212    <none>          9090/TCP                                                                                                                  2m
+    tracing                  ClusterIP      172.21.66.18     <none>          80/TCP                                                                                                                    2m
+    zipkin                   ClusterIP      172.21.205.179   <none>          9411/TCP                                                                                                                  2m
     {{< /text >}}
```

Comment:
I'm also seeing service graph on the list.

Reply:
fixed